module/VuFind/src/VuFind/Search/Factory/EITBackendFactory.php

```diff
@@ -51,7 +51,7 @@ class EITBackendFactory extends AbstractBackendFactory
     /**
      * Logger.
      *
-     * @var \Laminas\Log\LoggerInterface
+     * @var \Psr\Log\LoggerAwareInterface
```

Comment:
I think this still needs correction -- this is an actual logger, not a logger-aware service:

```suggestion
     * @var \Psr\Log\LoggerInterface
```